src/Assetic/Filter/JSqueezeFilter.php

```diff
@@ -46,7 +59,7 @@ public function filterLoad(AssetInterface $asset)
 
     public function filterDump(AssetInterface $asset)
     {
-        $parser = new \JSqueeze();
+        $parser = new $this->classname();
```

Comment:
It actually does work, yeah - i was a bit surprised too, but the unit tests do actually check this (I know because I got it wrong about ten times first, hence the "yakity sax" comment) and I also tested it on my oc8 test deployment (and made sure  had really been run and minified some files).